cli/cmdlineparser.cpp

```diff
@@ -96,6 +99,16 @@ static bool addPathsToSet(const std::string& fileName, std::set<std::string>* se
     return true;
 }
 
+static bool folderExist(const std::string &path)
```

Comment:
I think you can move this to lib/path.cpp .. add a function `Path::folderExist` instead.

Reply:
Moved function to Path module